broker/setup_broker/lsst/setup_broker.sh

```diff
@@ -52,10 +52,19 @@ define_GCP_resources() {
 }
 
 #--- GCP resources used directly in this script
+artifact_registry_repo=$(define_GCP_resources "cloud-run-services")
```

Comment:
If this repo is intended for use by our LSST broker only, then "lsst" should appear in its name. If this is really a Project setup task and the repo is intended for use by all of our brokers that operate in the same region, then this script should not delete it in the "teardown" section below.

Reply:
The repo should be deleted if it is a testing resource (i.e., it has a `testid` value). Will update to include the survey name

Reply:
> The repo should be deleted if it is a testing resource

Hmm, I think that still has the potential to cause problems since 2+ different broker testing instances can be running in the same project simultaneously. Perhaps we should treat this like the firewall where this script creates it but does not delete it. It will only actually be created once in a given project but since we don't have a different script for that here right now, this script can just go ahead and try to create it every time.

Reply:
Can you elaborate on the issue? 

If there are multiple testing instances in the same project, then the respective Artifact Registry repos should all have unique names due to the appended `testid`. I.e., the images in the Artifact Registry repo for different testing broker instances in the same project will be independent of one another. Deleting one repo, or manipulating its content, will not affect the content of a separate Artifact Registry repo in the same project.

Reply:
Oh sorry, yes a testid will keep them separate.